pkg/appgateserver/cmd/cmd.go

```diff
@@ -66,8 +67,9 @@ provided that:
 
 	// Cosmos flags
```

Comment:
I think after #256, we should not need to worry about mirroring the cosmos flags for the sake of being able to use cosmos CLI helpers because we won't need them anymore.

```suggestion
	// Cosmos flags
	// TODO_TECHDEBT(#256): remove unneeded cosmos flags.
```